.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - checkout
       - run:
           name: Acceptance Tests
-          command: 'yarn install && yarn acceptance src/test/acceptance/*.js'
+          command: "yarn install && yarn acceptance src/test/acceptance/*.js"
```

Comment:
random q - is there any reason for this diff? Do we prefer single quoted strings in circle ci config files?

Reply:
[prettier](https://prettier.io) does yml files, so these are the same code standard as the rest of front-end apps